core/modules/query/SelectStmt.h

```diff
@@ -112,14 +110,29 @@ class SelectStmt  {
     HavingClause& getHaving() { return *_having; }
     void setHaving(std::shared_ptr<HavingClause> h) { _having = h; }
 
+    // Helpers, for debugging
+
+    /** Return a string representation of the object
+     *
+     * Can be used for logging, of debugging
+     *
+     * @return a string representation of the object
+     */
+    std::string toString();
+
+    /** Return a string representation of the query::QueryTemplate
+     * object build from current object
+     *
+     * Can be used for logging, of debugging
+     *
+     * @return a string representation of the object
+     */
+    std::string toQueryTemplateString();
```

Comment:
No, it calls:

 getQueryTemplate().toString();

which is quite different, even if I don't understand the very details here

Please note that toString() returns a string which contains "FIXME" and isn't used in the code.